src/storage/ChainStorage.ts

```diff
@@ -1,11 +1,9 @@
 import { ChainID } from 'lib/eoaWallet'
 
-import { MMKVStorage } from 'storage/MMKVStorage'
-
-const ChainStorage = new MMKVStorage('chainStorage')
+import { MainStorage } from './MainStorage'
 
 export const getCurrentChainId: () => ChainID = () =>
-  ChainStorage.get('chainId') || 31
+  MainStorage.get('chainId') || 31
```

Comment:
Why is this ChainStorage being changed MainStorage? 🤔

Reply:
Because there was no need to do a separate instance just for `chainId`

Reply:
Have you tested to make sure that testnet and mainnet switch are working ok? 
And also wallet reset on testnet
And also wallet reset on mainnet

I ask because changing it could potentially alter the behavior of the wallet

Reply:
We're calling `getCurrentChainId` everytime when we need to fetch `chainId`

Reply:
Video proof

https://github.com/rsksmart/rif-wallet/assets/47615361/7e949e2e-fdd3-4f05-be84-e382e3698360